src/drivers/mod.rs

```diff
@@ -112,6 +112,9 @@ pub mod error {
 pub(crate) trait Driver {
 	/// Returns the interrupt number of the device
 	fn get_interrupt_number(&self) -> InterruptLine;
+
+	/// Returns the device driver name
+	fn get_name(&self) -> &'static str;
```

Comment:
This method is never called currently, right?

Reply:
But it might be used in the future to get rid of the additional map for names entirely.